features/util.py

```diff
@@ -65,6 +71,9 @@ def lxc_exec(
                 "stdout and stderr arguments may not be used "
                 "with capture_output."
             )
+        # stdout and stderr will be opened in text mode (by default they are
+        # opened in binary mode
+        kwargs["universal_newlines"] = True
```

Comment:
This looks like we will _always_ set `universal_newlines` if `capture_output` is given; should we instead only be setting it when the `text` parameter is `True`?

Reply:
Yes, I added it because if you do not set `universal_newlines = True` you'get this: `TypeError: a bytes-like object is required, not 'str'`.

Reply:
```suggestion
        if text:
            kwargs["universal_newlines"] = True
```